README.md

```diff
@@ -30,6 +30,7 @@ Kubernetes Fury Networking provides the following packages:
 | Package                    | Version  | Description                                                                                                                                          |
 | -------------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
 | [calico](katalog/calico)   | `3.25.0` | [Calico][calico-page] CNI Plugin. For cluster with `< 50` nodes.                                                                                     |
+| [cilium](katalog/cilium)   | `1.13.1` | [Cilium][cilium-page] CNI Plugin. For cluster with `< 50` nodes.                                                                                     |
```

Comment:
Is the "for clusters with < 50 nodes" also valid for Cilium or just a copy&paste error?

Reply:
I changed to < 200, since this article talks about what to do on more nodes: https://docs.cilium.io/en/v1.13/operations/performance/scalability/report/